### PR TITLE
Update Nexus URLs to Artifactory

### PR DESCRIPTION
**Pull request by lindluni**
Signed-off-by: Brett Logan <brett.t.logan@ibm.com>

### CODE REVIEW

File: scripts/npm_scripts/generateCerts.js [hunk 1]
```diff
@@ -26,7 +24,8 @@ module.exports.installAndGenerateCertsamd64 = async function() {
 };
 
 module.exports.installAndGenerateCertsMac = async function() {
-	await runShellCommand(`curl --create-dirs --output ${binariesPath}/${darwinTarFile} ${darwinBinaries}`, null);
+	await runShellCommand(`mkdir -p ${binariesPath}`, null);
+	await runShellCommand(`wget ${darwinBinaries} -P ${binariesPath}`, null);
```
Comment (bestbeforetoday) [inline scripts/npm_scripts/generateCerts.js]: `wget` does not exist on Macs by default, which is why the existing version uses `curl` instead

**Review by bestbeforetoday — CHANGES_REQUESTED** [review]

**Comment by btl5037** [comment]
I forgot I had changed this here. Switching to `wget` was meant to solve a problem we had been seeing with downloading the Fabric binaries when developers were in areas remote to GitHubs CDN. We had made the switch in Fabric for the bootstrap script and Gari had had us add `wget` as a prereq for Fabric. We quickly found out though, depending on which version of wget you have, they don't handle URLs generated from variables well. We need to revert our change in Fabric, and here as well. I'll switch it back to curl. Thanks for highlighting this.

File: scripts/npm_scripts/generateCerts.js (hunk 1 above)
Comment (btl5037) [reply scripts/npm_scripts/generateCerts.js]: I forgot I had changed this here. Switching to wget was meant to solve a problem we had been seeing with downloading the Fabric binaries when developers were in areas remote to GitHubs CDN. We had made the switch in Fabric for the bootstrap script and Gari had had us add wget as a prereq for Fabric. We quickly found out though, depending on which version of wget you have, they don't handle URLs generated from variables well. We need to revert our change in Fabric, and here as well. I'll switch it back to curl. Thanks for highlighting this.

Comment (btl5037) [reply scripts/npm_scripts/generateCerts.js]: Fixed, had to add the `-L` flag to handle redirects, tested to work

**Comment by btl5037** [comment]
@bestbeforetoday This is ready

**Comment by btl5037** [comment]
Tested on both Linux and Mac to make sure my changes to the gencerts work

**Review by bestbeforetoday — APPROVED** [review]

**Pull request merged**